### ISSUE TITLE
Document the HTML/XML report generator

### ISSUE THREAD

**Issue by JukkaL** [JukkaL/mypy#912]
I think that it's currently totally undcomented. Even though it's not very polished it could be useful, and we won't get bug reports if people don't even try it.

**Comment by o11c** [comment]
In readme or what? The website doesn't appear to live in any git repo.

I'm using it in https://github.com/o11c/lr-parsers though I haven't done much there since I started working on mypy.

**Comment by JukkaL** [comment]
The documentation lives under `docs/`. Hmm if you can't find it then it's probably not discoverable enough.

**Comment by o11c** [comment]
I must confess I'm used to ignoring any folder called `doc/` or `docs/` because it's always obsolete. `--help` and `pydoc3` are usually more useful.

Still would like to see source for the mypy website proper.

**Comment by JukkaL** [comment]
Yeah, publishing mypy website sources is somewhere on my to-do list but it's a big mess and I might be too embarrassed to publish it without some cleaning up.
